recipes/zmqpp/all/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
```suggestion
set(CMAKE_CXX_STANDARD 11)
```

Why do you need to enforce c++ standard?

Reply:
zmqpp recommends C++11 compliant compiler. And builds the library and its examples in C++11 mode.

Reply:
You can remove it, it's already inserted in upstream. https://github.com/zeromq/zmqpp/blob/4.2.0/CMakeLists.txt#L18

Reply:
Ok